sdcm/sct_config.py

```diff
@@ -1693,6 +1700,8 @@ class SCTConfiguration(dict):
                     'eks_cluster_version', 'eks_nodegroup_role_arn'],
     }
 
+    test_required_params = {"perf-regression-gradual": ["perf_gradual_throttle_steps", "perf_gradual_threads"]}
```

Comment:
took me a while to get it, I'm not sure I'm happy with this logic of counting on the name of the test for those things assuming extra required parameters.

we are o.k. without it, in all of there cases

I would suggest to drop it, and do such a thing on it's own, it's not mandatory for this test

Reply:
Because "perf_gradual_throttle_steps" and "perf_gradual_threads" are required for perf-regression-gradual test only

Reply:
> took me a while to get it, I'm not sure I'm happy with this logic of counting on the name of the test for those things assuming extra required parameters.
> 
> we are o.k. without it, in all of there cases
> 
> I would suggest to drop it, and do such a thing on it's own, it's not mandatory for this test

It is mandatory for this test. I can validate this later, but I am not sure that it is better to start and configure the test and fail it on mandatory params

Reply:
I understand why we would want such a validation per test,
but it's something we should define/do in a separate task, where we can design how per test configuration should be define / validated

I don't think it's mandatory validation for this test case, and we can take it out.

Reply:
Do add validation of those parameters in the test?

Reply:
not that critical to have validation, what would happen if you don't have it ? test would fail ? test would succeed without doing anything ?


it's a test we are looking at the data/results even if it's passing, we won't be easily miss if it's broken

Reply:
Should fail

Reply:
We can live with it (like in any other test specific configuration we have without validation)

Reply:
removed